lib/poker_hands/hand_rankings/royal_flush.rb

```diff
@@ -1,5 +1,14 @@
 class RoyalFlush
   def check(hand)
-    true
+    royal_flush_ranks = ['A', 'K', 'Q', 'J', '10']
+    ranks_in_hand = []
+    
+    return false if hand.any? { |card| card.suit != hand.first.suit}
```

Comment:
Are we passing in a `Hand` object or an array of `Card` objects?

Reply:
In my code there is no such thing as a Hand object, in this case hand is an array of Card objects as you say. Does that need to be more clear, or do you ask because I had previously talked about implementing a Hand class (that I decided not to do).